docs/getting-started/kubernetes.mdx

```diff
@@ -0,0 +1,106 @@
+
+import K8sIngressModifyHeadersExample from "/examples/k8s/http-request-headers.mdx";
+import K8sGatewayModifyHeadersExample from "/examples/k8s/http-request-headers-gateway.mdx";
+
+import K8sIngressRedirectExample from "/examples/k8s/http-request-redirect.mdx";
+import K8sGatewayRedirectExample from "/examples/k8s/http-request-redirect-gateway.mdx";
+
+import StaticDomainKubernetesExample from "/examples/k8s/http-static-domain.mdx";
+import StaticDomainKubernetesGatewayExample from "/examples/k8s/http-static-domain-gateway.mdx";
+
+import StaticDomainKubernetesPathExample from "/examples/k8s/http-static-domain-path.mdx";
+import StaticDomainKubernetesGatewayPathExample from "/examples/k8s/http-static-domain-gateway-path.mdx";
+
+# Quickstart: Kubernetes Operator
+
+This quickstart will use the [ngrok Kubernetes Operator](https://github.com/ngrok/kubernetes-ingress-controller) to put an application running on a Kubernetes Cluster on the internet and add some basic policies.
+
+## What is the ngrok Kubernetes Operator?
+
+The ngrok Kubernetes Operator is the best way to use ngrok if your applications run as Services in a Kubernetes cluster. Some of the benefits of the operator are:
+
+- The Operator can be easily installed on a cluster via Helm and is compatible with the most recent versions of Kubernetes.
+- The Operator supports both [Ingress CRDs](https://kubernetes.io/docs/concepts/services-networking/ingress/) as well as the new [Gateway API](https://gateway-api.sigs.k8s.io/) using the same installation.
+- When deployed to a cluster, the Operator allows developers to self-service public endpoints to their apps and services using a shared ngrok account.
+- The Operator manages the creation of ngrok resources, leaving developers to focus on traffic and routing patterns.
+- The Operator supports the configuration of resources like custom domains, modules, and policies.
+- The Operator supports all the cloud benefits of ngrok, including our global points of presense, GSLB, and traffic inspector.
+
+## Step 1: Install Kubernetes Operator
+
+Installing the ngrok Operator is easy using Helm.
+
+<Tabs groupId="k8s" queryString="k8s-install">
+	<TabItem value="ingress" label="Ingress Controller" default>
+		<K8sIngressInstallExample />
+	</TabItem>
+	<TabItem value="gatewayAPI" label="Gateway API">
+		<K8sGatewayAPIInstallExample />
+	</TabItem>
+</Tabs>
+
+<br />
+
+## Step 2: Connect your service to an Edge
+
+This would create an Edge on with the Domain `example.ngrok.app` that routes traffic to the Kubernetes service `example-service` running on port `80`.
+
+<Tabs groupId="k8s" queryString="k8s-basic">
+	<TabItem value="ingress" label="Ingress Controller" default>
+		<StaticDomainKubernetesExample />
+	</TabItem>
+	<TabItem value="gatewayAPI" label="Gateway API">
+		<StaticDomainKubernetesGatewayExample />
+	</TabItem>
+</Tabs>
+
+## Step 3: Route Traffic based on a path
```

Comment:
are we capitalizing the titles or not? seems like a mix throughout the PR. can we standardize?